.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -39,9 +39,9 @@ Here are some guidelines to help the review process go smoothly.
    features or make changes out of the scope of those requested by the reviewer
    (doing this just add delays as already reviewed code ends up having to be
    re-reviewed/it is hard to tell what is new etc!). Further, please do not
-   rebase your branch on master/force push/rewrite history, doing any of these
+   rebase your branch on main/force push/rewrite history, doing any of these
```

Comment:
@kkraus14 Looks like the changes to occurences of `master` in `0.18` have not been pushed so this PR brings the changes from `master` as this is branch is based on `master`. 

Also, I targeted this to `0.18` release since this is a typo in docs. Let me know if we want to move this to `0.19`.